Cargo.toml

```diff
@@ -134,11 +134,10 @@ members = [
 	"primitives/transaction-pool",
 	"primitives/trie",
 	"primitives/wasm-interface",
-	"test/utils/chain-spec-builder",
```

Comment:
Why is that removed?

Reply:
it depends on `node-cli` but no one, not even `node` nor `node-template` uses it. I suspect it is a left over from a previous refactor that move the logic into the cli....

Reply:
It's a standalone utility that's why nothing depends on it.

Reply:
I think we should rather move it to `bin`.

Reply:
rebased so I can properly move it into `/bin/utils`. as we have that now, I also moved `subkey` there... 

one could argue they should actually be in `/bin/node/utils` as they are bound to `node`...